src/praisonai-code/praisonai_code/cli/configuration/resolver.py

```
@@ -453,6 +501,163 @@ def _load_project_config(self) -> Optional[Dict[str, Any]]:
+        """Fetch managed config from a URL, fail-soft to the cached copy.
+
+        Never blocks a run on the network: a short timeout is used and any
+        failure falls back to the last good on-disk cache, else ``None`` so the
+        managed layer is simply skipped (offline == local-only behaviour).
+        """
+        cache_path = self._managed_cache_path(url)
+        data: Optional[Dict[str, Any]] = None
+        try:
+            import urllib.request
+
+            with urllib.request.urlopen(url, timeout=timeout) as resp:  # nosec B310
```

Ruff (0.15.20)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)